packages/ra-core/src/routing/resolveRedirectTo.ts

```diff
@@ -0,0 +1,42 @@
+import { To } from 'react-router-dom';
+
+/**
+ * @deprecated use useCreatePath instead
```

Comment:
Same

Reply:
Same answer: we don't remove it to avoid a more painful upgrade.